server/src/app/redisutil.js

```diff
@@ -76,28 +78,44 @@ module.exports = (function() {
       });
     },
 
-    loadPollResults: function(pollId) {
+    loadPollResults: async function(pollId) {
       let rKey = pollId + "R";
       let aKey = pollId + "A";
-      this.redisGet(aKey).then(aValue => {
-        let answers = JSON.parse(aValue);
-        let results = [];
-        for (let i = 0; i < answers.length; i++) {
-          let result = new Result();
-          result.option = answers[i];
-          result.index = i;
-          result.count = 0;
-          results.push(result);
-        }
-        this.redisGet(rKey).then(rValue => {
-          let counts = JSON.parse(rValue);
-          for (let j = 0; j < counts.length; j++) {
-            results[counts[j]].count++;
-          }
-          console.log(results);
-          return results;
-        });
-      });
+      let aValue = await this.redisGet(aKey);
+      let answers = JSON.parse(aValue);
+      let results = [];
+      for (let i = 0; i < answers.length; i++) {
+        let result = new Result();
+        result.option = answers[i];
+        result.index = i;
+        result.count = 0;
+        results.push(result);
+      }
+      let rValue = await this.redisGet(rKey);
+      let counts = JSON.parse(rValue);
+      for (let j = 0; j < counts.length; j++) {
+        results[counts[j]].count++;
+      }
+      return results;
+      // this.redisGet(aKey).then(aValue => {
```

Comment:
Remove this or do you want to leave it?

Reply:
I left it just in case we had to go back to it, but if you don't think so I can remove it.